firefox-ios/Client/Frontend/Widgets/OneLineTableViewCell.swift

```diff
@@ -90,8 +90,12 @@ class OneLineTableViewCell: UITableViewCell,
         super.layoutSubviews()
         updateReorderControl()
 
+        // Position the accessory at the trailing edge of the cell, accounting for safe area and padding
         if let accessoryView {
-            accessoryView.frame.origin.x = frame.width - accessoryView.frame.width - UX.accessoryViewTrailingPadding
+            accessoryView.frame.origin.x = frame.width
+            - accessoryView.frame.width
+            - UX.accessoryViewTrailingPadding
+            - safeAreaInsets.right
```

Comment:
Quick question about RTL languages wouldn't this need to be left ?

Reply:
That is a great callout - since this issue is mostly unrelated to this change (the accessory views are currently still always pinned to the right side of the cell, regardless of the fact that we are now accounting for the safe area), I have created FXIOS-11149